tsconfig.json

```diff
@@ -33,14 +32,13 @@
     ],
     "preserveConstEnums": false,
     "skipDefaultLibCheck": true,
-    "sourceMap": false,
     // "strictBindCallApply": true,
     // "strictFunctionTypes": true,
     // "strictNullChecks": true,
     // "strictPropertyInitialization": true,
     "stripInternal": true,
     "target": "es5",
-    "outDir": "dist/esm",
+    "outDir": "ts-dist/",
```

Comment:
Changes look great. Sourcemaps and tests both work. Minor suggestion: the folder name `ts-dist` is too close to "dist" for my taste, given that the former is only for internal use while the latter is published to npm. Possible to pick another name with "test" or "internal" in the name and "dist" not in it?

Reply:
I've moved this to ./tsc-out/ so it's very clear what is putting files there.